src/sweeper/__main__.py

```diff
@@ -74,27 +78,30 @@ def main():
         closet.append(EmptyTest(args['--workspace'], args['--table-name']))
         closet.append(MetadataTest(args['--workspace'], args['--table-name']))
 
-    reports = execute_sweepers(closet, args['--try-fix'], args['--change-detect'])
+    reports = execute_sweepers(closet, args['--try-fix'], args['--change-detect'], log)
```

Comment:
**question**: does the log need to be passed down when you can request it in the different files e.g. `log = logging.getLogger('sweeper')` or did you make this choice because it's a method in the same file?

Reply:
I think I did it because it was in the same file and I was getting an error without it.  That might've been in another spot, so I'll see if I can get it to run without passing into `execute_sweepers`

Reply:
Yeah, it seems to crash if I don't pass the log into the `execute_sweepers` function:
![image](https://user-images.githubusercontent.com/38330621/128389409-e82dc8ec-f086-42ee-8c2e-033b42dcb983.png)

Reply:
that error makes it look like you're using log without `log = logging.getLogger('sweeper')`

Reply:
Is it because I call that in a function?

line 51:
`log = setup_logging(args['--save-report'], args['--scheduled'])`

the function:
```
def setup_logging(save_report, scheduled):
    
    logger = logging.getLogger('sweeper')
    logger.setLevel(logging.INFO) 
    .
    .
    .
    return logger
```

Reply:
@steveoh are there any outstanding questions for this one?  I think we once we merge it we can confidently set up the scheduled task to run based on change detection.